### PR TITLE
Script release preparations

### PR DESCRIPTION
**Pull request by mfalkvidd**
This automates steps 1-3 of
https://github.com/mysensors/MySensors/wiki/Library-releases

### CODE REVIEW

File: .mystools/prepare-release.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+if [ ! -d mysensors-release ]; then
+	git clone -o upstream git@github.com:mysensors/MySensors.git mysensors-releas
```
Comment (fallberg) [inline .mystools/prepare-release.sh]: typo: mysensors-releas

Comment (mfalkvidd) [reply .mystools/prepare-release.sh]: Good catch. I have pushed an update.

**Comment by fallberg** [comment]
With the development branch now properly in sync with master, we hopefully won't have to do this very complicated merging anymore. It should just be a regular merge that generates no conflicts from now on.

**Comment by mfalkvidd** [comment]
Ok. Lets close this and update the wiki instead. Not complicated is good.

**Pull request closed without merging**